### DISCUSSION TITLE
Add reason field for pausing an alert

### DISCUSSION THREAD

**Discussion by vpereira** [grafana/grafana#40849] [Alerting]
When putting an alert on Pause, it would be nice if we had a "reason" field which could be set as mandatory or not, depending on the policy in place.

**Comment by torkelo** [comment]
the new alerting system adds a new silence mechanic where you can define a time range, and message for why alerts where silcenced 

https://grafana.com/docs/grafana/latest/alerting/unified-alerting/silences/

**Reply by vpereira** [reply]
Hi @torkelo 

Thank you for your help!

I'm however running Grafana 8.1.2 but still don't see the Silences option while hovering over the "Bell".. There is any missing configuration step to enable it?

**Reply by JohnnyQQQQ** [reply]
Yes, you need to opt-in. Here is everything explained in detail https://grafana.com/docs/grafana/latest/alerting/unified-alerting/opt-in/

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
